### ISSUE TITLE
Setup Swagger

### ISSUE THREAD

**Issue by gala-dan** [GalateaRaj/fuse-starter-java#220]
Issue Description
--
Setup Swagger within FUSE. Very useful to demonstrate how to document a RESTful API for consumers of the API.

Design
--
See http://www.baeldung.com/swagger-2-documentation-for-spring-rest-api for the basics. We'd need to consider any security implications and at the very least highlight where they'd be addressed.

Documentation Changes
--
Update documentation to provide address to raw Swagger and Swagger UI endpoints.

Test Evidence
--
Once implemented we should be able to navigate to the raw and UI endpoints.
![image](https://user-images.githubusercontent.com/23705265/45897977-9f526c80-bda6-11e8-9898-f00def4d6282.png)

Validation in Develop
-- 
[Proof issue is resolved in Develop. Should be complete before Issue is closed]

**Comment by gala-dan** [comment]
Note I got this working locally by:

1. Adding the Swagger dependencies to the pom.xml (with the below exclusion in place for both)
2. Adding a SwaggerConfig class
3. Adding the code from the "Configuration Without Spring Boot" section of the Baeldung article to MvcConfig

Exclusions:
```
<exclusions>
    <exclusion>
        <groupId>com.google.guava</groupId>
	<artifactId>guava</artifactId>
    </exclusion>
</exclusions>
```

**Comment by j-barber** [comment]
Merged under PR #241
